test/jvm/src/main/scala/zio/test/TestClockPlatformSpecific.scala

```diff
@@ -60,11 +60,11 @@ trait TestClockPlatformSpecific { self: TestClock.Test =>
             final case class Running(cancelToken: Scheduler.CancelToken, end: Instant) extends State
             final case class Scheduling(end: Instant)                                  extends State
 
-            val state = new AtomicReference[State](Scheduling(initial))
-            val latch = new CountDownLatch(1)
+            val localState = new AtomicReference[State](Scheduling(initial))
```

Comment:
Was this required by your changes? It seems that you're fixing an unrelated issue here, but please correct me if not!

Reply:
CI / test (3.2.0, 17, JVM) failed without this change